Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -1083,3 +1087,41 @@ private func qualtricsConfigData() -> QualtricsConfigData {
     .compact()
   )
 }
+
+private func shouldGoToLandingPage() -> Bool {
+  guard AppEnvironment.current.currentUser == nil else {
```

Comment:
Won't `AppEnvironment.current.currentUser` be nil if the user is logged out? In which case we might be unnecessarily activating an experiment for users that have already seen the landing page?

We might want to check the ubiquitous store as the first step so we can break early if we know the user has already seen the landing page, in which case we don't need to check the optimizely experiment.

Reply:
Hmm good point. I will change that 👍